src/app.js

```diff
@@ -181,6 +202,7 @@ class Messenger extends EventEmitter {
       .then((session) => this.saveSession(session));
   }
 
+  // WIP
```

Comment:
oh, I was marking that these methods are vestigial from the beauty lens bot and they're kinda dead code at this point but we want to bring them back eventually. I didn't want to start yak shaving to bring them up to "code" 😸 to keep this PR smaller. But I wanted to leave a mark to remind myself to stop trying to work on them.